resources/idlharness.js

```diff
@@ -257,7 +257,8 @@ IdlArray.prototype.internal_add_idls = function(parsed_idls, options)
 
     parsed_idls.forEach(function(parsed_idl)
     {
-        if (parsed_idl.type == "interface" && parsed_idl.partial)
+        if (parsed_idl.partial
+            && ["interface", "dictionary"].find(t => parsed_idl.type == t))
```

Comment:
Optional simplification: `["interface", "dictionary"].includes(parsed_idl.type)` would do the trick here too.

Reply:
lol. tried 'contains' and fell back to something I knew. Changed.